shell/common/rasterizer.h

```diff
@@ -430,6 +430,22 @@ class Rasterizer final : public SnapshotDelegate {
   ///
   std::optional<size_t> GetResourceCacheMaxBytes() const;
 
+  //----------------------------------------------------------------------------
+  /// @brief      Makes sure the raster task runner and the platform task runner
+  /// are merged
+  ///
+  /// @attention  If raster and platform task runners are not the same or not
+  /// merged. This method
```

Comment:
could we fix the indentation?

Reply:
It seems the formatter did something crazy. Fixed.